README.md

```diff
@@ -396,11 +322,6 @@ Cognita makes it really easy to switch between parsers, loaders, models and retr
 
 -   Register the vectordb under `backend/modules/vector_db/__init__.py`
 
-### Rerankers:
```

Comment:
Why remove this?

Reply:
Earlier there was a reranker file that handled the logic, now it's routed by infinity service. Can't change much here so removed it.